src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykScanListenerLS.kt

```diff
@@ -69,42 +69,44 @@ class SnykToolWindowSnykScanListenerLS(
 
         val settings = pluginSettings()
 
-
         // display Security issues
-        val securityIssues = snykResults
-            .map { it.key to it.value.filter { issue -> issue.additionalData.isSecurityType } }
-            .toMap()
+        val securityIssues =
+            snykResults
+                .map { it.key to it.value.filter { issue -> issue.additionalData.isSecurityType } }
+                .toMap()
+
         displayIssues(
             enabledInSettings = settings.snykCodeSecurityIssuesScanEnable,
-            filterTree =  settings.treeFiltering.codeSecurityResults,
+            filterTree = settings.treeFiltering.codeSecurityResults,
             snykResults = securityIssues,
             rootNode = this.rootSecurityIssuesTreeNode,
             securityIssuesCount = securityIssues.values.flatten().distinct().size,
+            fixableIssuesCount = securityIssues.values.flatten().distinct().count { it.hasAIFix() },
```

Comment:
Why only security issues? I think in VSCode, all issues are counted.

Reply:
I chose to only do this for Code Security issues (could do Code Quality too maybe) because of what I said [here](https://github.com/snyk/snyk-intellij-plugin/pull/536#discussion_r1624068884). Do you mean I should do the same for Code Quality too or something else?

Reply:
But shouldn't we have all Snyk Code results covered by the UI/UX for ignores? I think it should be added for Code Quality, too, as this is also covered by the consistent ignores scope, right? Also, it would make it similar to how it works in VSCode, IIRC.

Reply:
Yes good point, I'll do that 👍